src/pydata_sphinx_theme/logo.py

```diff
@@ -78,4 +78,4 @@ def copy_logo_images(app: Sphinx, exception=None) -> None:
                 f"The {kind} logo path '{path_image}' looks like a Sphinx template; "
                 "please provide a static logo image."
             )
-        copy_asset_file(path_image, staticdir)
+        copy_asset_file(str(Path(app.srcdir) / path_image), staticdir)
```

Comment:
on second look, I'm not sure this is correct.  If the logo path is supposed to be relative to `conf.py` then shouldn't this be:

```suggestion
        copy_asset_file(str(Path(app.confdir) / path_image), staticdir)
```

Reply:
I believe the reason is that it's already referencing  `(Path(app.srcdir) / path_image)` on line 73:

```python
        if not (Path(app.srcdir) / path_image).exists():
```

We likely want to use a variable and reuse it to make sure there is no inconsistencies.

Reply:
and `copy_asset_file(path_image, staticdir)` was wrong as afaict, this was referencing path relative to CWD, and just silently do nothing if the source does not exist.